src/Discord.Net.Core/Entities/Messages/EmbedBuilder.cs

```diff
@@ -44,6 +45,26 @@ public EmbedBuilder WithUrl(string url)
             Url = url;
             return this;
         }
+        public EmbedBuilder WithThumbnailUrl(string thumbnailUrl)
+        {
+            ThumbnailUrl = thumbnailUrl;
+            return this;
+        }
+        public EmbedBuilder WithImageUrl(string imageUrl)
+        {
+            ImageUrl = ImageUrl;
+            return this;
+        }
+        public EmbedBuilder WithTimestamp()
```

Comment:
This is unclear. should be more obvious that it's setting Timestamp to Now.
`WithNowTimestamp()` maybe? Unless someone has a better name.